draft-ietf-keytrans-architecture.md

```diff
@@ -400,6 +400,94 @@ Alice                  Transparency Log                  Manager
 Transparency Log to the Manager. Rejected requests are blocked." }
 
 
+# Combining Logs
+
+There are many cases where it makes sense to operate multiple cooperating log
+instances, for example:
+
+- A service provider may decide that it's prudent to rotate its cryptographic
+  keys, or migrate to a new deployment mode. They can do this by creating a new
+  log instance with new cryptographic keys, operating under a new
+  deployment mode if desired, and migrating their data from the old log to the
+  new log while users are able to query both.
+- A service provider may choose to operate multiple logs to improve their
+  ability to scale or provide higher availability.
+- A federated system may allow each participant in the federation to operate
+  their own log for their own users.
+
+Client implementations should generally be prepared to interact with multiple
+logs simultaneously. In particular, clients SHOULD namespace any configuration
+or state related to a particular log, such that information related to different
+logs do not conflict.
+
+When multiple logs are used, all users in the system MUST have a consistent
+policy for executing Search, Update, and Monitor queries against the logs in a
```

Comment:
I wonder if an example policy might make this clearer... It reads as " the service provider should have a policy of not being duplicitous"...but if that is detectable, what does this MUST really unlock?

Reply:
This "MUST" doesn't affect the service provider, it's saying all of the users need to behave in a consistent way. If they don't, it gets very hard to reason about whether security guarantees are met

Reply:
So this is a policy suggestion for "clients" ? maybe client is a better word than user here.... saying "must be consistent" without further clarity remains triggering for me, but your answer to makes sense to me.